Userland/Libraries/LibWeb/HTML/Window.cpp

```diff
@@ -604,15 +603,7 @@ bool Window::has_transient_activation() const
 {
     // The transient activation duration is expected be at most a few seconds, so that the user can possibly
     // perceive the link between an interaction with the page and the page calling the activation-gated API.
-    auto transient_activation_duration = 5;
-
-    // AD-HOC: Due to resource limitations on CI, we cannot rely on the time between the activation timestamp and the
-    //         transient activation timeout being predictable. So we allow tests to indicate when they want the next
-    //         check for a user gesture to succeed.
-    if (m_internals && m_internals->bypass_next_transient_activation_test()) {
-        m_internals->set_bypass_next_transient_activation_test(false);
-        return true;
-    }
+    static constexpr HighResolutionTime::DOMHighResTimeStamp transient_activation_duration = 5000;
```

Comment:
Should we add a constructor for DOMHighResTimeStamp that takes an AK::Duration so we can name the units here?

Reply:
The current definition of it is:
```c++
using DOMHighResTimeStamp = double;
```
I think changing that would be a pretty big change that affects e.g. IDL

Reply:
oof ok. That's unfortunate.

Reply:
Maybe add `_ms` to the name?